reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -326,6 +330,9 @@ def init(self):
         # Should check that cluster is not already initialized.
         # Maybe use `verify_components()` or `get()` each component?
 
+        if traefik is True:
+            self.initialize_traefik()
```

Comment:
I think that we should fail here if `traefik` could not be installed. For instance now I forgot to do `helm init` and I get the following:
```console
$ reana-cluster -f reana_cluster/configurations/reana-cluster-dev.yaml init --traefik
Error: could not find tiller
[ERROR] Traefik initialization failed
 Command '['helm', 'install', 'stable/traefik', '--namespace', 'kube-system--values', '/Users/rodrigdi/code/reanahub/reana-cluster/reana_cluster/configurations/helm/traefik/minikube.yaml']' returned non-zero exit status 1..
REANA cluster is initialised.
```
And the cluster is initialised either way:
```console
$ kubectl get pods
NAME                                   READY   STATUS              RESTARTS   AGE
db-6d845f7ff9-4mwjx                    0/1     ContainerCreating   0          14s
job-controller-56c984749-wlfch         1/1     Running             0          14s
message-broker-5bf98cc4f9-g2747        1/1     Running             0          14s
server-66f4d75c47-kz5d5                2/2     Running             0          14s
wdb-6b794c6855-zwtph                   0/1     ContainerCreating   0          14s
workflow-controller-7d457fc4cf-5gcrb   2/2     Running             0          14s
```
But traefik is not installed.

We could do something like:
```console
$ reana-cluster -f reana_cluster/configurations/reana-cluster-dev.yaml init --traefik
Tiller could not be found, please run helm init to initialise it.
```
What do you think?